Separate `problem` and `solution` review steps in RFCs (#2584)
sourcegraph/handbook: content/company-info-and-process/communication/rfcs/index.md

We have a set of status labels we’ve found helpful in the past:

- **WIP**: The author is still drafting the RFC and it’s not ready for review.

> - **Reviewing problem**: The Reviewing problem label is used when the RFC has a firm problem statement and is ready to confirm that the problem definition is correct.

Reviewer: ```suggestion
- **Reviewing problem**: The Reviewing problem label is used when the RFC has a firm problem statement, and reviews should focus on confirming that the problem definition is correct.
```
